portfolio/src/main/webapp/script.js

```diff
@@ -110,9 +110,49 @@ function handleSubmit(e) {
 /** Creates a map and adds it to the page. */
 function createMap() {
   const map = new google.maps.Map($('#map')[0], {
-    center: { lat: 37.422, lng: -122.084 },
-    zoom: 16,
+    center: { lat: 41.8719, lng: 12.5674 },
+    zoom: 2,
+    mapTypeId: 'hybrid',
   });
+
+  const places = [
```

Comment:
No need to do this for now. But could be a good candidate for moving to its own file to treat this as a static data source. Especially if more than 1 file wants to reference this.

Reply:
I'll keep that in mind, thanks!